mitosheet/mitosheet/tests/streamlit/test_streamlit.py

```diff
@@ -62,7 +62,7 @@
         'return type of analysis',
         [df1], {'return_type': 'analysis'},
         (
-            RunnableAnalysis('', None, '', [])
+            RunnableAnalysis('', None, '', [], 0)
```

Comment:
Huh -- what is this extra param?

Reply:
Oo this was a test for of the RunnableAnalysis params and it was missing one, so figured I would add it.